docs/user/kafka/usage.rst

```diff
@@ -65,6 +66,8 @@ Parameter                            Type    Description
                                              See :ref:`kafka_ssi`
 ``kafka.serialization.lazy``         Boolean Use lazy deserialization of features. This may improve processing load at
                                              the expense of slightly slower query times
+``kafka.schema.registry.url``        String  URL to a confluent schema registry server, used to read Confluent schemas for kafka topics
+                                             serialized with the Confluent avro serializer. Experimental, read-only, record timestamps required.
```

Comment:
I believe timestamps were included in 0.10, right? Could we update this to be 'requires Kafka 0.10 or later' instead of 'record timestamps required'? Otherwise it's not clear what is actually needed. Same thing above with the serialization type.

Reply:
Actually I guess we are defaulting to system time now if there is no timestamp, so we can probably remove the bit about timestamps entirely